### PR TITLE
fix: media preview bug when zero size, close #764

### LINKED ISSUE #764: Image is grey in Pictures View
### Describe the bug

As shown in the video, in picture view, when you click on the details page of an item, the picture cannot be displayed and is always gray.

### Feed Info

rsshub://voronoiapp/latest

### Reproduction Video

https://github.com/user-attachments/assets/328d1a27-b57f-4d70-a17c-a8ec2baeb763

### App Type

Web

### Operating System

Windows 11

### Browser Info

_No response_

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.

### Contributions

- [x] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

### PR DESCRIPTION
**Pull request by cesaryuan**
### Description

Close #764

### Linked Issues

#764

### CODE REVIEW

File: apps/renderer/src/components/ui/media/preview-media.tsx [hunk 1]
```diff
@@ -317,11 +319,15 @@ const FallbackableImage: FC<
             blurhash && !isLoading ? "duration-500 ease-in-out animate-in fade-in-0" : "",
             props.className,
           )}
-          style={{
-            maxHeight: `min(100%, ${Number.parseInt(props.height as string)}px)`,
-            maxWidth: `min(100%, ${Number.parseInt(props.width as string)}px)`,
-            ...props.style,
-          }}
+          style={
+            height * width === 0
```
Comment (Innei) [inline apps/renderer/src/components/ui/media/preview-media.tsx:323]: If height or width got `Nan` there will got some issue, `NaN * NaN !== 0`

**Review by Innei — CHANGES_REQUESTED** [review]

File: apps/renderer/src/components/ui/media/preview-media.tsx (hunk 1 above)
Comment (cesaryuan) [reply apps/renderer/src/components/ui/media/preview-media.tsx]: done

File: apps/renderer/src/components/ui/media/preview-media.tsx [hunk 2]
```diff
@@ -317,11 +319,17 @@ const FallbackableImage: FC<
...
           )}
-          style={{
-            maxHeight: `min(100%, ${Number.parseInt(props.height as string)}px)`,
-            maxWidth: `min(100%, ${Number.parseInt(props.width as string)}px)`,
-            ...props.style,
-          }}
+          style={
+            Number.isNaN(height) || Number.isNaN(width) || height === 0 || width === 0
+              ? {
+                  ...props.style,
```
Comment (Innei) [inline apps/renderer/src/components/ui/media/preview-media.tsx:325]:
```suggestion
      props.style
```

**Review by Innei — APPROVED** [review]

**Review by Innei — APPROVED** [review]

**Pull request merged**